IRremoteESP8266.cpp

```diff
@@ -856,7 +856,7 @@ long IRrecv::decodeNEC(decode_results *results) {
     results->decode_type = NEC;
     return DECODED;
   }
-  if (irparams.rawlen < 2 * NEC_BITS + 4) {
+  if (irparams.rawlen < 2 * NEC_BITS + 2 ) { // buf len is 64+(2)+1? and not 2*32+4
```

Comment:
Your comment doesn't match the code.
i.e. 2 * NEC_BITS + 2 == 2 * 32 + 2 = 66 .. where as the comment is 64+2+1 = 67

But as for the correct value. I'm not sure. I'm not fully up to speed on the decoding aspects, so take the next bit with a grain of salt.

The original value of 68 seems correct to me.
1 * initial space (skipped by the offset) +  2 (HDR mark + HDR space) + 2 * 32 bits of NEC "data" (a mark & space pair) + a trailing space.  Thus 1 + 2 + 64 + 1 == 68. Or am I missing something?